paper.md

```diff
@@ -147,7 +151,7 @@ Some of the major changes in each module of ``VIP`` are summarized below:
     [@Lafreniere:2007] was added;
     - an annular version of the non-negative matrix factorization algorithm 
     is now available [@Lee:1999; @Gomez:2017];
-    - besides median-ADI, the `medsub` routine now also supports median-SDI. 
+    - the `medsub` routine now also supports median-SDI. 
```

Comment:
I think it's important to mention median-ADI because it is not trivial what `medsub` stands for. Would this sound better in English: 
> the `medsub` routine now also supports median-SDI, in addition to median-ADI.